src/Resources/SimpleModelItemRestResource.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Rhubarb\RestApi\Resources;
+
+use Rhubarb\Crown\Context;
+use Rhubarb\Stem\Models\Model;
+
+class SimpleModelItemRestResource extends ModelItemRestResource
```

Comment:
ModelItemRestResource doesn't exist - was that meant to be ModelRestResource or is there a class missing?